src/lib/wallet/providers/onekey.ts

```diff
@@ -0,0 +1,16 @@
+import type { AddressInfo, Transaction, WalletStrategy } from "./shared";
```

Comment:
We should only show either Onekey OR Alby

And Onekey can implement all the same methods that Alby supports